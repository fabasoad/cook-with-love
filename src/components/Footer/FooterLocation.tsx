import * as React from 'react'
import { useTranslation } from 'react-i18next'

export default function FooterLocation() {
  const { t } = useTranslation()
  return (
    <div className="col">
      <i className="fa fa-map-marker-alt fa-2x"></i>
      <p>{t('cook-with-love-contact-city')}</p>
    </div>
  )
}
