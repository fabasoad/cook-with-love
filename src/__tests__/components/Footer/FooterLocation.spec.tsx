/// <reference types="jest" />
import { shallow, ShallowWrapper } from 'enzyme'
import * as React from 'react'
import FooterLocation from '../../../components/Footer/FooterLocation'
import { useTranslation } from '../../__mocks__/react-i18next'

let tMock

describe(FooterLocation.name, () => {
  beforeAll(() => {
    tMock = useTranslation().t
  })

  test('should render FooterLocation correctly', () => {
    const wrapper: ShallowWrapper = shallow(<FooterLocation />)
    expect(wrapper).toMatchSnapshot()
    expect(tMock).toHaveBeenCalledTimes(1)
    expect(tMock).toHaveBeenCalledWith('cook-with-love-contact-city')
  })
})
