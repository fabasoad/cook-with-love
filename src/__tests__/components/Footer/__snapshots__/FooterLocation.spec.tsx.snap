// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLocation should render FooterLocation correctly 1`] = `
<div
  className="col"
>
  <i
    className="fa fa-map-marker-alt fa-2x"
  />
  <p>
    cook-with-love-contact-city
  </p>
</div>
`;
